### PR TITLE
Tags/aj 136 create tag with all fields

### PR DESCRIPTION
**Pull request by JorgeRCV**

### CODE REVIEW

**Review by Raymundogs — APPROVED** [review]

File: src/test/java/cucumber/steps/ApiSteps.java [hunk 1]
```diff
@@ -70,8 +70,8 @@ public void executesRequest(final String apiMethod) throws IllegalAccessExceptio
                 .method(ApiMethod.valueOf(apiMethod))
                 .build();
         ApiManager.execute(apiRequest, apiResponse);
-        IFeature featureResponse = apiResponse.getBody(featureFactory.getFeature(this.featureName).getClass());
-        scenarioContext.setBaseEnvironment(String.format("%s_id", featureName), featureResponse.getIdentifier());
+//        IFeature featureResponse = apiResponse.getBody(featureFactory.getFeature(this.featureName).getClass());
```
Comment (GustavoZHA) [inline src/test/java/cucumber/steps/ApiSteps.java]: Delete comment, these two lines are not necessary?

File: src/test/java/cucumber/steps/SetTagBodyStep.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        trashList.add(body.get("name"));
+        scenarioContext.setTrash("Tags Trash", trashList);
+        JSONObject jsonBody = new JSONObject();
+        JSONObject tagBody = new JSONObject();
+        for (String tagComponent: body.keySet()) {
+            tagBody.put(tagComponent, body.get(tagComponent));
+        }
+        jsonBody.put("tag", tagBody);
+        apiRequestBuilder.body(jsonBody.toString());
+        System.out.println(jsonBody.toString());
```
Comment (GustavoZHA) [inline src/test/java/cucumber/steps/SetTagBodyStep.java]: Delete sout

File: src/test/java/cucumber/steps/AddTagsStep.java [hunk 3]
```diff
@@ -33,25 +35,24 @@ public void tagsBulkAdd(int amount) {
...
-                    + "}");
+            tagBody.put("name", tagName + i);
+            apiRequestBuilder.body(jsonBody.put("tag", tagBody).toString());
             apiRequest = apiRequestBuilder.build();
-            tagsTrashList.add(tagName);
-            scenarioContext.setTrash("Tags Trash", tagsTrashList);
             ApiManager.execute(apiRequest, apiResponse);
             apiResponse.getResponse().then().log().body();
+            tagsTrashList.add(tagName);
+
```
Comment (GustavoZHA) [inline src/test/java/cucumber/steps/AddTagsStep.java]: Delete blank line

File: src/test/resources/features/tags/tags_tests.feature [hunk 4]
```diff
@@ -0,0 +1,11 @@
+Feature: Create Tags
+
+  @DeleteTag
```
Comment (GustavoZHA) [inline src/test/resources/features/tags/tags_tests.feature]: The tag is correct? the purpose of scenay is to create a tag

File: src/test/java/cucumber/steps/SetTagBodyStep.java [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+    public SetTagBodyStep(ApiRequestBuilder apiRequestBuilder, ApiResponse apiResponse, SoftAssert softAssert) {
+        this.apiRequestBuilder = apiRequestBuilder;
+        this.apiResponse = apiResponse;
+        this.softAssert = softAssert;
+    }
+
+    @When("I set the tags body with following values:")
+    public void setTagBody(final Map<String, String> body) {
+        List<String> trashList =  new ArrayList<>();
+        System.out.println(body.get("name"));
```
Comment (GustavoZHA) [inline src/test/java/cucumber/steps/SetTagBodyStep.java]: Delete sout

File: src/test/java/cucumber/runner/RunTests.java [hunk 6]
```diff
@@ -71,6 +71,7 @@ public void createSpace() throws JsonProcessingException {
                 .build();
         ApiManager.execute(apiRequest, apiResponse);
         scenarioContext.setBaseEnvironment("space_id", apiResponse.getBody(Space.class).getId());
+
```
Comment (GustavoZHA) [inline src/test/java/cucumber/runner/RunTests.java]: Delete blank line

File: src/test/java/cucumber/runner/RunTests.java [hunk 7]
```diff
@@ -30,14 +30,14 @@
 import org.testng.annotations.BeforeTest;
 
 @CucumberOptions(
-        features = {"src/test/resources/features"},
+        features = {"src/test/resources/features/tags"},
```
Comment (GustavoZHA) [inline src/test/java/cucumber/runner/RunTests.java]: Delete /tags

File: src/test/java/cucumber/hooks/TagsHooks.java [hunk 8]
```diff
@@ -67,6 +68,8 @@ public void deleteTags() {
             apiRequest = apiRequestBuilder.build();
             ApiManager.execute(apiRequest, apiResponse);
             apiResponse.getResponse().then().log().body();
+            System.out.println("deleting " + tagName);
```
Comment (GustavoZHA) [inline src/test/java/cucumber/hooks/TagsHooks.java]: Delete sout

**Review by GustavoZHA — APPROVED** [review]

**Pull request merged**